velox/expression/tests/SparkExpressionFuzzerTest.cpp

```diff
@@ -45,8 +45,10 @@ int main(int argc, char** argv) {
   // The following list are the Spark UDFs that hit issues
   // For rlike you need the following combo in the only list:
   // rlike, md5 and upper
+  // regexp_replace: https://github.com/facebookincubator/velox/issues/8438
```

Comment:
Is this for regexp_replace or all regexp_xxx functions? Move the comment right above regexp_xxx (L50) and remove "regexp_replace:"

```
  std::unordered_set<std::string> skipFunctions = {
      "regexp_extract",
      // https://github.com/facebookincubator/velox/issues/8438
      "regexp_replace",
```

Reply:
It's just for regexp_replace. As far as I can tell, the other regexp functions do not leverage the cache issues we discovered in #8438 so it's not relevant to them. Will move comment

Reply:
> the other regexp functions do not leverage

Is this intentional? We need to make sure no function is compiling regex per row.